sycl/unittests/Extensions/CommandGraph/Update.cpp

```diff
@@ -399,3 +399,14 @@ TEST_F(WholeGraphUpdateTest, EmptyNode) {
   auto GraphExec = Graph.finalize(experimental::property::graph::updatable{});
   GraphExec.update(UpdateGraph);
 }
+
+TEST_F(CommandGraphTest, CheckFinalizeBehaviour) {
```

Comment:
Peak nitpicking
```suggestion
TEST_F(CommandGraphTest, CheckFinalizeBehavior) {
```

Reply:
Good catch, I try and train out the british spellings but it still slips through every now and then!